test/powershell/Modules/Microsoft.PowerShell.Utility/Update-List.Tests.ps1

```diff
@@ -0,0 +1,29 @@
+Describe "Update-List tests" -Tags CI {
+    BeforeEach {
+        $list = [System.Collections.Generic.List``1[int]]::new()
```

Comment:
You don't need ` ``1` - I'm guessing you copied this from an old test that was written before we had better generics support.

Reply:
Yup, will change.